dclab/kde/base.py

```diff
@@ -169,6 +170,7 @@ def get_contour_lines(self, xax="area_um", yax="deform", xacc=None,
             contour line.
         levels: list of floats
             The interpolated density levels for corresponding quantiles levels.
```

Comment:
The word "interpolated" is not correct. I would write "The density levels corresponding to `quantiles`".